test/test_mcmc.py

```diff
@@ -36,6 +36,9 @@ def potential_fn(z):
     init_params = jnp.array(0.)
     if kernel_cls is SA:
         kernel = SA(potential_fn=potential_fn, dense_mass=dense_mass)
+    elif kernel_cls is BarkerMH:
+        # TODO: fix dense_mass once BarkerMH supports it
```

Comment:
i'll add dense_mass support soon so i guess it's ok if we have duplicate tests for a few days